lua/octo/pickers/telescope/entry_maker.lua

```diff
@@ -694,6 +704,64 @@ function M.gen_from_octo_actions()
   end
 end
 
+function M.gen_from_notification()
+  local make_display = function(entry)
+    if not entry then
+      return nil
+    end
+
+    local columns = {
+      entry.obj.unread == true and icons.notification[entry.kind].unread or icons.notification[entry.kind].read,
+      { "#" .. (entry.obj.subject.url:match "/(%d+)$" or "NA") },
+      { string.sub(entry.obj.repository.full_name, 1, 50), "TelescopeResultsNumber" },
+      { string.sub(entry.obj.subject.title, 1, 100) },
+    }
+
+    local displayer = entry_display.create {
+      separator = " ",
+      items = {
+        { width = 2 },
+        { width = 12 },
+        { width = math.min(#entry.obj.repository.full_name, 50) },
+        { width = math.min(#entry.obj.subject.title, 100) },
+      },
+    }
```

Comment:
The other pickers (pr list, issue list, discussion list, and search) use columns:  

- number
- Symbol
- title

Would it make sense to consolidate to that ordering? Maybe squeeze the repo before title

Reply:
im also not sure if that order makes sense for the notifications...just from a UI perspective